.github/ci.sh

```diff
@@ -158,7 +162,7 @@ install_system_deps() {
   install_yasm &
   wait
   export PATH=$PWD/$BIN:$PATH
-  echo "$PWD/$BIN" >> $GITHUB_PATH
+  echo "$PWD/$BIN" >> "$GITHUB_PATH"
```

Comment:
Why do you quote everything like this?

Reply:
Irrational levels of paranoia combined with following all the warnings shellcheck provides result in lots of quoting. It may technically be unnecessary in this case, but there's no reason not to; unless minimizing diffs is a valid concern, I suppose